Formula/qt.rb

```diff
@@ -24,39 +24,35 @@ class Qt < Formula
     sha256 "25c4a693c787860b090685ac5cbeea18128d4d6361eed5b1bfed1b16ff6e4494" => :mojave
   end
 
-  keg_only "Qt 5 has CMake issues when linked"
+  keg_only "Qt has CMake issues when linked"
 
+  depends_on "cmake" => [:build, :test]
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+
   depends_on xcode: :build
-  depends_on macos: :sierra
+  depends_on macos: :mojave
```

Comment:
Add these dependencies.
  depends_on "assimp"
  depends_on "double-conversion"
  depends_on "freetype"
  depends_on "icu4c" # Qt needs icu's utilities
  depends_on "jpeg"
  depends_on "libb2"
  depends_on "libpng"
  depends_on "pcre2"
  depends_on "pkg-config"
  depends_on "zstd"
In addition, what about the sql plugins?

Reply:
Qt SQL supported DBs are:
  DB2 (IBM)
    FEATURE_sql_db2:BOOL
  InterBase
    FEATURE_sql_ibase:BOOL
  MySql
    FEATURE_sql_mysql:BOOL
  OCI (Oracle)
    FEATURE_sql_oci:BOOL
  ODBC
    FEATURE_sql_odbc:BOOL
  PostgreSQL
    FEATURE_sql_psql:BOOL
  SQLite
    FEATURE_sql_sqlite:BOOL